docs/configuration_options.md

```diff
@@ -279,18 +306,19 @@
+| **<a href="#user-content-scylla_rsyslog_setup" name="scylla_rsyslog_setup">scylla_rsyslog_setup</a>**  | Configure rsyslog on Scylla nodes to send logs to monitoring nodes | N/A | SCT_SCYLLA_RSYSLOG_SETUP
+| **<a href="#user-content-events_limit_in_email" name="events_limit_in_email">events_limit_in_email</a>**  | Limit number events in email reports | 10 | SCT_EVENTS_LIMIT_IN_EMAIL
+| **<a href="#user-content-data_volume_disk_num" name="data_volume_disk_num">data_volume_disk_num</a>**  | Number of additional data volumes attached to instances<br>if data_volume_disk_num > 0, then data volumes (ebs on aws) will be<br>used for scylla data directory | N/A | SCT_DATA_VOLUME_DISK_NUM
 | **<a href="#user-content-data_volume_disk_type" name="data_volume_disk_type">data_volume_disk_type</a>**  | Type of addtitional volumes: gp2|gp3|io2|io3 | N/A | SCT_DATA_VOLUME_DISK_TYPE
 | **<a href="#user-content-data_volume_disk_size" name="data_volume_disk_size">data_volume_disk_size</a>**  | Size of additional volume in GB | N/A | SCT_DATA_VOLUME_DISK_SIZE
 | **<a href="#user-content-data_volume_disk_iops" name="data_volume_disk_iops">data_volume_disk_iops</a>**  | Number of iops for ebs type io2|io3|gp3 | N/A | SCT_DATA_VOLUME_DISK_IOPS
-| **<a href="#user-content-scylla_bench_version" name="scylla_bench_version">scylla_bench_version</a>**  | A valid tag under the scylla bench repo: https://github.com/scylladb/scylla-bench | N/A | SCT_SCYLLA_BENCH_VERSION
-| **<a href="#user-content-nosqlbench_image" name="nosqlbench_image">nosqlbench_image</a>**  | A docker image of NoSQLBench | N/A | SCT_NOSQLBENCH_IMAGE
-| **<a href="#user-content-raid_level" name="raid_level">raid_level</a>**  | Configure RAID0 or RAID5 on instance | N/A | SCT_RAID_LEVEL
-| **<a href="#user-content-bare_loaders" name="bare_loaders">bare_loaders</a>**  | Don't install anything but collectd to the loaders during cluster setup | False | SCT_BARE_LOADERS
-| **<a href="#user-content-run-db-node-benchmarks" name="run_db_node_benchmarks">run_db_node_benchmarks</a>**  | Run benchmarks on db nodes before the test | false | SCT_RUN_DB_NODE_BENCHMARKS
+| **<a href="#user-content-run_db_node_benchmarks" name="run_db_node_benchmarks">run_db_node_benchmarks</a>**  | Flag for running db node benchmarks before the tests | N/A | SCT_RUN_DB_NODE_BENCHMARKS
+| **<a href="#user-content-nemesis_selector" name="nemesis_selector">nemesis_selector</a>**  | nemesis_selector gets a list of "nemesis properties" and filters IN all the nemesis that has<br>ALL the properties in that list which are set to true (the intersection of all properties).<br>(In other words filters out all nemesis that doesn't ONE of these properties set to true)<br>IMPORTANT: If a property doesn't exist, ALL the nemesis will be included. | N/A | SCT_NEMESIS_SELECTOR

```

Comment:
Addition of the `nemesis_selector` to the doc is unrelated to the goal of this commit. Please, make it in a separate PR.

Reply:
this document is created automatically, filtering for unwanted stuff is not easy - I rather update it manually then.

Reply:
Do the update of the docs in its own commit.

Is just because we are missing those updates when configuration names changed or added

Reply:
Done in separate commit.